org.mfek.MFEKglif.yml

```diff
@@ -0,0 +1,75 @@
+app-id: org.mfek.MFEKglif
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+cleanup: 
+  - /sdk
+build-options:
+  append-path: /app/sdk/rust-nightly/bin
+  env: 
+    CARGO_HOME: /run/build/MFEKglif/cargo
+    SKIA_BINARIES_URL: "file:///run/build/MFEKglif/skia-binaries.tar.gz"
+    FORCE_SKIA_BINARIES_DOWNLOAD: "1"
+finish-args: 
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --filesystem=home
+command: MFEKglif
+modules:
+  - name: MFEKglif
+    buildsystem: simple
+    build-commands:
+      - cd rust && ./install.sh --prefix=/app/sdk/rust-nightly --without=rust-docs
+        --without=rust-docs-json-preview --disable-ldconfig --verbose
+      - cp ./resources/MFEKglif.desktop ./${FLATPAK_ID}.desktop
+      - cp ./resources/MFEKglif.png ./${FLATPAK_ID}.png
+      - desktop-file-edit ./${FLATPAK_ID}.desktop --set-key=Icon --set-value=org.mfek.MFEKglif.png
+      - cargo --offline fetch --manifest-path Cargo.toml --verbose
+      - cargo --offline build --release --features=sdl2-static
+      - install -Dm755 ./target/release/MFEKglif -t /app/bin/
+      - install -Dm644 ./${FLATPAK_ID}.metainfo.xml -t /app/share/metainfo/
+      - install -Dm644 ./${FLATPAK_ID}.desktop -t /app/share/applications/
+      - install -Dm644 ./${FLATPAK_ID}.png -t /app/share/icons/hicolor/512x512/apps/
+    sources:
+      - type: git
+        url: https://github.com/MFEK/glif.git
+        commit: 8764b3e40a6dc219f3dc7db1429c58dff9795840
+      - type: file
+        only-arches:
+          - x86_64
+        url: https://github.com/rust-skia/skia-binaries/releases/download/0.62.0/skia-binaries-8cf4841aefdb295709e9-x86_64-unknown-linux-gnu-gl.tar.gz
+        dest-filename: skia-binaries.tar.gz
+        sha256: e4806ae8566b1cc1e7692d9c5f0e0a32754a07591db6de884c107adea1922b7a
+      - type: file
+        only-arches:
+          - aarch64
+        path: skia-binaries-0.62.0-aarch64-unknown-linux-gnu-gl.tar.gz
```

Comment:
This shouldn't be here. If they don't supply aarch64 builds, and you can't build from source, then build for x86_64 only.

Shipping packages built on your PC isn't a good idea. The trusted source of this is the rust-skia organisation.

Reply:
I spent some time trying to chase down building this lib from source offline but ultimately the process is too confusing for me to figure out. There is an open pull request https://github.com/rust-skia/rust-skia/pull/757 that will add these binaries to the distributed ones, and once it is merged, we can either hopefully get them to release binaries for the version we're using, or look into updating the library to the latest. Are there any legitimate blockers that prevent this from being merged as is? The things I saw discussed seemed to be more suggestions than anything. And anything else that wasn't I addressed.